spec/yoti/dynamic_share_service/extension/location_constraint_extension_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'Yoti::DynamicSharingService::LocationConstraintExtension' do
+  describe '.type' do
+    let :ext do
+      Yoti::DynamicSharingService::LocationConstraintExtension
+        .builder
+        .build
+    end
+    it 'has the location constraint extension type' do
+      expect(ext.type).to eql Yoti::DynamicSharingService::LocationConstraintExtension::EXTENSION_TYPE
+    end
+  end
+
+  describe '.builder' do
+    describe '.with_latitude' do
```

Comment:
Doesn't look we have test coverage for when latitude, longitude, radius & uncertainty are floats, and when they are outside of the range. If we allow for floats, we need to make sure we have tests for it 👍  and also checking that the exceptions are raised appropriately. This can be done in a similar way to the other SDKs if we can have parameterized tests